src/test/java/com/google/api/codegen/testdata/python_doc_main_library.baseline

```diff
@@ -0,0 +1,876 @@
+============== file: library/library_service_api.py ==============
```

Comment:
They do -- see `python_doc_message_library.baseline`. During one of the refactorings, there was a behavior change so that a single Java test corresponds to multiple baseline files -- one baseline file for each template.